docker-compose.yml

```diff
@@ -18,6 +18,13 @@ services:
       - 8443:443
     volumes:
       - ./src/backend:/var/www/html
+    healthcheck:
+      test: [CMD, curl, -f, http://localhost/up]
+      start_period: 30s
+      interval: 10s
+      timeout: 5s
+      retries: 3
+
```

Comment:
I don't think it adds a lot of value in the local compose setup, but just makes the compose file more complex.

The real benefit I would see is the use case as a Kubernetes [liveness and readiness probe](https://kubernetes.io/docs/concepts/configuration/liveness-readiness-startup-probes/). However, this is a separate task.